File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    arr = list(range(n+1))  
    
    arr[0] = 0
    arr[1] = 1        
        
    for i in range(2, n+1):
        arr[i] = arr[i-1] + arr[i-2]
                
    return arr[n] % 1234567

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

int solution(int n) {
    
    int a = 0;
    int b = 1;
    
    for(int i = 1; i < n; i++)
    {
        int tmp = a + b;
        a = b;
        b = tmp % 1234567;
    }    
        
    return b;
}